src/App/Pages/Accounts/LoginPageViewModel.cs

```diff
@@ -149,8 +149,7 @@ public async Task InitAsync()
                     Email = await _stateService.GetRememberedEmailAsync();
                 }
                 var deviceIdentifier = await _appIdService.GetAppIdAsync();
-                // TODO uncomment to enable login with device
-                //IsKnownDevice = await _apiService.GetKnownDeviceAsync(Email, deviceIdentifier);
+                IsKnownDevice = await _apiService.GetKnownDeviceAsync(Email, deviceIdentifier);
```

Comment:
just in case, is this ok to be enabled?

Reply:
It needs to be enabled in order for QA to test it here in the branch. But when the time comes to be merged, i should be good to be enabled.